bundles/ranvier-commands/commands/look.js

```diff
@@ -1,17 +1,63 @@
 'use strict';
 
+const leftPad = require('left-pad');
+
 module.exports = (srcPath) => {
   const Broadcast = require(srcPath + 'Broadcast');
   const CommandParser = require(srcPath + 'CommandParser').CommandParser;
   const Item = require(srcPath + 'Item');
   const ItemType = require(srcPath + 'ItemType');
   const Logger = require(srcPath + 'Logger');
 
+  function getCompass(player) {
+    const room = player.room;
+
+    const exitMap = new Map();
+    exitMap.set('east', 'E');
```

Comment:
That's true - an array may make more sense here. I don't think custom exits could exist with this, they would not be rendered by the compass currently. Would have to be accounted for.